beacon_node/src/cli.rs

```diff
@@ -422,6 +422,17 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        failure. Be extremely careful with this flag.")
                 .takes_value(true)
         )
+        .arg(
+            Arg::with_name("terminal-block-hash-epoch-override")
+                .long("terminal-block-hash-epoch-override")
+                .value_name("EPOCH")
+                .help("Used to coordinate manual overrides to the TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH \
+                       parameter. This flag should only be used if the user has a clear understanding \
+                       that the broad Ethereum community has elected to override the terminal PoW block. \
+                       Incorrect use of this flag will cause your node to experience a consensus
+                       failure. Be extremely careful with this flag.")
+                .takes_value(true)
```

Comment:
I think we should make a "requires" dependency between both `"terminal-block-hash-override"` and `"terminal-block-hash-epoch-override"` (i.e. you can't supply one without the other).

I can't see how they'd be effective without each other and it would be nice to fail early.